database/truncate.sh

```diff
@@ -0,0 +1,3 @@
+source .env
```

Comment:
Nice way of resetting the database, in a way that can be rolled back if need be.